release-notes/0.1.2.rst

```diff
@@ -0,0 +1,7 @@
+0.1.2 (2023-12-04)
```

Comment:
All these release notes files were manually created by me going through your GitHub releases. That was only to backfill historical releases. New releases won't need that.